pkg/oc/cli/admin/release/release.go

```diff
@@ -0,0 +1,19 @@
+package release
+
+import (
+	"github.com/spf13/cobra"
+
+	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	"k8s.io/kubernetes/pkg/kubectl/genericclioptions"
+)
+
+func NewCmd(f kcmdutil.Factory, parentName string, streams genericclioptions.IOStreams) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "release",
+		Short: "Tools for managing the OpenShift release process",
```

Comment:
Hmmm... does this belong to oc? Seems like it should be a separate tool, imo.

Reply:
It's going to go into OC for now, we may extract it later, but it's effectively something every admin will use.

Reply:
I.e. this will be the command you use to update a cluster:

```
oc adm release upgrade
oc adm release status
oc adm release mirror
```

and more